app/models/article.rb

```diff
@@ -1,3 +1,4 @@
 class Article < ApplicationRecord
   validates :title, :body, :author, presence: true
+  belongs_to :user
```

Comment:
I would write the association like this `belongs_to :author, class_name: 'User'`

Then you can remove the author attribute from the article model